.github/workflows/build-git-annex-windows.yaml

```diff
@@ -86,6 +88,11 @@ jobs:
         ./Build/BuildVersion > dist/build-version
       working-directory: git-annex
 
+    - name: Capture build-version
+      id: build-version
+      run: echo "::set-output name=version::$(cat dist/build-version)"
```

Comment:
sorry to be a pain, but IIRC that dist/build-version which annex produces is "subopimal":
```shell
$> Build/BuildVersion
8.20201008-ga3bb6f235% 
```
so it is not sortable at all.  That is why we should not use it for "our" (this one is what what @joeyh wants/uses) version, which should be based on the output of `git describe`:
```shell
$> git describe
8.20201007-145-ga3bb6f235
```
but with a slight tune up (my personal convention I guess)
```
$> git describe | sed -e 's,-,+git,' 
8.20201007+git145-ga3bb6f235
```
which makes it a bit more obvious what is that version about (`git` still understands it since looks only for `-g` suffix with hexsha).
And then artifact name should include architecture. E.g. in http://datasets.datalad.org/datalad/packages/windows/ I added `_x64` since felt that is what windows folks got used to, although in case of a debian package should be `_amd64`  (it is no longer "amd" but that is what debian uses, heh).

Reply:
Is using the output of `uname -p` as the architecture acceptable, or should it be specifically `_x64` for Windows, `_amd64` for Linux, and ... what? ... for macOS?

Reply:
well, for "debian" (current only Linux) if to be specific, should be output of 
```
$> dpkg --print-architecture
amd64
```
ran within that singularity env. And if I download that .zip with artifacts, I see that we do have version already nicely set for that .deb:
```
$> unzip git-annex-debianstandalone-packages.zip 
Archive:  git-annex-debianstandalone-packages.zip
  inflating: git-annex-build.log     
  inflating: git-annex-standalone_8.20201007+git145-ga3bb6f235-1~ndall+1_amd64.deb  
...
```
no tune up needed for it at all, and may be you could just take that version altogether from the name of the .deb file?

For Windows and Mac -- I have no personal preference really. `uname -p` under git shell on windows tells me "unknown" though.